lib/health_monitor/providers/database.rb

```diff
@@ -7,12 +7,29 @@ module Providers
     class DatabaseException < StandardError; end
 
     class Database < Base
+      class Configuration
+        attr_accessor :databases
+
+        def initialize
+          @databases = [nil]
+        end
+      end
+
       def check!
-        # Check connection to the DB:
-        ActiveRecord::Migrator.current_version
+        configuration.databases.each do |database|
+          ActiveRecord::Base.establish_connection(database)
+        end
```

Comment:
I don't think this works. It might also be better to use all configured connections with something like 

```rb
ActiveRecord::Base.connection_handler.all_connection_pools.map { |cp| cp.connection.current_database }
```

However, for me this resulted in 4 connection pools, two for each of the two databases I have configured.

Reply:
Why do you think that `ActiveRecord::Base.establish_connection` won't be sufficient?

Reply:
I think it only connect to the primary/default database.

Reply:
Also it looks like you can only pass it an env name or a full configuration
https://api.rubyonrails.org/classes/ActiveRecord/ConnectionHandling.html#method-i-establish_connection

Reply:
Tried it just now with purposely broken config for second db

```rb
ActiveRecord::Base.establish_connection
#<ActiveRecord::ConnectionAdapters::ConnectionPool:0x00007f7176891ce8 

ActiveRecord::Base.establish_connection(:secondary)
#<ActiveRecord::ConnectionAdapters::ConnectionPool:0x00007f7176891ce8 

ActiveRecord::Base.connection_handler.all_connection_pools.map { |cp| cp.connection.current_database }
# => ActiveRecord::NoDatabaseError: We could not find your database: crap. Which can be found in the database configuration file located at config/database.yml.
```

Reply:
I'm not sure that this method is appropriate either. During my tests, after fixing `database.yml`, `ActiveRecord::Base.connection_handler.all_connection_pools` only had a single connection to `database1`. I will try to find the time to look for a more robust/explicit method.

Reply:
I'm not sure but this may be because there needs to be a model that points to the second database.

Reply:
I'll give it a shot and also check how replicas are treated.

Reply:
Hey @HarlemSquirrel.

Could you take a look at the latest commit?